snuba/migrations/policies.py

```diff
@@ -1,10 +1,13 @@
 from abc import ABC, abstractmethod
+from datetime import datetime, timedelta
 
 from snuba.migrations.groups import get_group_loader
 from snuba.migrations.runner import MigrationKey, Runner
 from snuba.migrations.status import Status
 from snuba.utils.registered_class import RegisteredClass
 
+MAX_REVERT_TIME_WINDOW_HRS = 24
```

Comment:
Lets define this in settings so that we only need setting changes to happen if we need to change it